### PR TITLE
Fix leading nans bug in _aggregate

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [x] Code changes are covered by tests
- ~New functions added to `__init__.py`~
- ~API.rst is up to date, along with other sphinx docs pages~
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog

See #244

### CODE REVIEW

File: rdtools/normalization.py [hunk 1]
```diff
@@ -620,6 +620,9 @@ def _aggregate(time_series, target_frequency, max_timedelta, series_type):
     max_interval_nanoseconds = max_timedelta.total_seconds() * 10.0**9
 
     gap_mask = t_diffs > max_interval_nanoseconds
+    if time_series.index[0] != union_index[0]:
```
Comment (mdeceglie) [inline rdtools/normalization.py:623]: would using `.sum(min_count=1)` in line 605 above fix the bug? Might offer a cleaner fix.

Comment (mdeceglie) [reply rdtools/normalization.py:623]: I retract this suggestion

**Comment by kanderso-nrel** [comment]
Discussed with @mdeceglie offline: one notebook changed, there was a ~3day spike at the beginning of the normalized energy series caused by leading nans in the POA timeseries.  No relevant changes in the other two notebooks.

File: docs/sphinx/source/changelog/v2.0.4.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+**************************
+v2.0.4 (MONTH DAY, YEAR)
+**************************
+
+Bug Fixes
+---------
+* Fix bug related to leading NaN values with
+  :py:func:`~rdtools.normalization.energy_from_power` (:issue:`244`, :pull:`245`)
+
```
Comment (mdeceglie) [inline docs/sphinx/source/changelog/v2.0.4.rst:9]: Let's capture the notebook change in the log as well.

**Review by mdeceglie — APPROVED** [review]
Looks good, thanks for finding and patching this!

**Pull request merged**